lib/nethealth/nethealth.go

```diff
@@ -508,7 +509,7 @@ func (s *Server) processAck(e messageWrapper) error {
 		s.WithFields(logrus.Fields{
 			"peer_name": peer.name,
 			"peer_addr": peer.addr,
-			"id":        pkt.Seq,
+			"id":        peer.echoCounter,
```

Comment:
Should this be converted to uint16 as well? Otherwise what's in the logs won't match what's actually being compared. We can also include the overall "real" counter e.g.

```go
  "counter": peer.echoCounter,
  "seq": uint16(peer.echoCounter),
```